vignettes/customise.Rmd

```diff
@@ -89,8 +89,8 @@ You can find the correct height by running `$(".navbar").outerHeight()` in the [
 ### bslib variables
 
 Instead of picking a complete theme, you can tweak fonts and colours individually using bslib variables.
-[bslib](https://rstudio.github.io/bslib/) is an R package that wraps sass, the tool that Boostrap uses to produce CSS from a special language called [scss](https://sass-lang.com).
-The primary advantage of scss over CSS is that it's more programmable, so you can have a few key bslib variables that affect appearance of many HTML elements.
+[bslib](https://rstudio.github.io/bslib/) is an R package that wraps scss, the tool that Boostrap uses to produce CSS from a special language called [scss](https://sass-lang.com).
```

Comment:
The tool is called sass (https://cloud.r-project.org/web/packages/sass/index.html) and the language is called scss.